frontend/src/pages/Facharbeit/FacharbeitStudentListPage.tsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * Adminansicht mit Schülerliste für die Facharbeit.
+ */
+/*eslint-disable */
+
+import React from 'react';
+import TableListComponent from '../../components/ReusableComponents/TableListComponent';
+import ComponentToggleButton from '../../components/Buttons/ComponentToggleButton';
+import './FacharbeitenPageStyles.css'
+import CreatePDFButton from "../../components/Buttons/CreatePDFButton";
+
+function handleActivateComponentButton (){
+  alert("komponente aktiviert")
+}
+
+const handleGeneratePDF = () => {
+  alert('pdf soll jetzt generiert werden')
+}
+
+function facharbeit() {
```

Comment:
die Komponente bitte so nennen wie die Datei

Reply:
Diese Komponente, bzw. die Tabelle ist auch noch nicht richtig responsiv

Reply:
Die Tabelle hab ich jetzt ein bisschen angepasst, aber grundsätzlich ist es sehr schwer eine so flexible komponente im vornerein schon responsiv zu gestalten. ich weiß ja nicht was da jedesmal reinkommt. sind es nur 4 spalten mit nur text? oder 5 textspalten, ein textfeld und eine checkbox? Um die responsivität muss sich jeweils derjenige kümmern, der die tabelle instanziiert, da nur er oder sie weiß was da am ende drin steht

Reply:
Achso ändert sich die Tabelle nochmal? Dachte die bleibt jetzt so und ist fertig

Reply:
Habe es mir gerade nochmal angeschaut und die Tabelle ist leider immer noch abgeschnitte und drin scrollen kann ich zwar, aber nicht bis ganz links, so dass ich alles sehe
![responsivität](https://user-images.githubusercontent.com/70637384/134413698-c49baf20-e658-4d7f-b3bb-2b4fefc6bda8.PNG)
e.

Reply:
Ne, das soll ja eine wiederverwendbare komponete sein.

und der horizontal scroll funktioniert jetzt. darfst die tabelle nicht in ne flexbox packen, sonst kackt die horizontal scrollbar ab aus irgendeinem grund